enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -52,53 +92,24 @@ type PlacementSpec struct {
 	// +optional
 	DecisionStrategy DecisionStrategy `json:"decisionStrategy,omitempty"`
 }
-
-// Subset of the created placementDecision with label.
-type DecisionGroup struct {
-	// Label to be added to the created placement Decisions
-	Label string  `json:"labels,omitempty"`
-
-  // LabelSelector to select clusters subset by label.
-	// +optional
-	ClusterSelector metav1.LabelSelector `json:"clusterSelector,omitempty"`
-}
-
-// Group the created placementDecision into decision groups based on the number of clusters per decision group.
-type DecisionStrategy struct {
-	// Define subsets of the created placement decisions and add labels to the placementDecision.
-	// +optional
-	DecisionGroups []DecisionGroup `json:"decisionGroups,omitempty"`
-
-	// ClusterPercentagePerDecisionGroup is a percentage of the total selected clusters that will divide the clusters to groups.
-	// ex; for a total 100 clusters selected, ClusterPercentagePerDecisionGroup equal to 20% will divide the placement decision to 5 groups each group should have 20 clusters.
-	// +kubebuilder:validation:Enum=None;5%;10%;15%;20%;25%;30%;50%;100%
-	// +kubebuilder:default:=None
-	// +optional
-	ClusterPercentagePerDecisionGroup string `json:"clusterPercentagePerDecisionGroup,omitempty"`
-
-	// NumberOfClustersPerDecisionGroup is the max number of clusters per decision group. If ClusterPercentagePerDecisionGroup is used the NumberOfClustersPerDecisionGroup will be ignored.
-	// +kubebuilder:default:=100
-	// +optional
-	NumberOfClustersPerDecisionGroup int32 `json:"numberOfClustersPerDecisionGroup,omitempty"`
-}
-
 ```
 
 Adding DecisionGroupStatus to the PlacementStatus API 
+
 ```go
 // Present decision groups status based on the DecisionStrategy definition.
 type DecisionGroupStatus struct {
 	// Present the decision group index. If there is no decision strategy defined all placement decisions will be in group index 0
```

Comment:
Today, the placement decision starts from index1, for example, placement `demo` will generate placementdecision with name `demo-decision-1`. My question is should the DecisionGroupIndex also starts from 1 to be consistent.

Reply:
Not a big deal, but better to start with index 0 cause the groups count and index start with 1 might be miss leading.

Reply:
Yes, after consideration, the group starting from 0 makes sense. Placement decision name is just for readability, don't need to be considered together.